### PR TITLE
CLOUDP-199262: fix local dev containers stop/start

### PR DESCRIPTION
**Pull request by tibulca**
_Jira ticket:_ CLOUDP-199262

### CODE REVIEW

File: internal/cli/atlas/deployments/setup.go [hunk 1]
```diff
@@ -177,15 +177,18 @@ func (opts *SetupOpts) configureMongod(ctx context.Context, keyFileContents stri
...
-				`echo $KEYFILECONTENTS > $KEYFILE && \
-                   chmod 400 $KEYFILE && \
-                   python3 /usr/local/bin/docker-entrypoint.py \
-                                          --transitionToAuth \
-                                          --dbpath $DBPATH \
-                                          --keyFile $KEYFILE \
-                                          --replSet $REPLSETNAME \
-                                          --setParameter "mongotHost=$MONGOTHOST" \
-                                          --setParameter "searchIndexManagementHostAndPort=$MONGOTHOST"`,
+				`if [ ! -f $KEYFILE ]
```
Comment (tibulca) [inline internal/cli/atlas/deployments/setup.go:180]: without this check, stopping and starting back mongod will fail (as keyfile is already there)

File: internal/cli/atlas/deployments/setup.go [hunk 2]
```diff
@@ -177,15 +177,18 @@ func (opts *SetupOpts) configureMongod(ctx context.Context, keyFileContents stri
...
+                 then
+                     echo $KEYFILECONTENTS > $KEYFILE
+                     chmod 400 $KEYFILE
+                 fi
+                 python3 /usr/local/bin/docker-entrypoint.py \
```
Comment (blva) [inline internal/cli/atlas/deployments/setup.go:185]: [q] do we need to capture the python3 dependency as well somewhere or do all OS come with it by default?

File: internal/cli/atlas/deployments/setup.go (hunk 1 above)
Comment (blva) [reply internal/cli/atlas/deployments/setup.go:180]: nice!

**Review by blva — APPROVED** [review]

File: internal/cli/atlas/deployments/setup.go (hunk 1 above)
Comment (gssbzn) [reply internal/cli/atlas/deployments/setup.go:180]: these file are becoming a bit gnarly, you should consider moving them to dedicates `.sh` files so they are checked with shellcheck and include them via file embedding, see https://github.com/mongodb/mongodb-atlas-cli/blob/fdb8b0cf8b9a87be29095b2d4edfc56fccdd652d/tools/chocolateypkg/chocolateypkg.go#L68C1-L69

File: internal/cli/atlas/deployments/setup.go (hunk 2 above)
Comment (blva) [reply internal/cli/atlas/deployments/setup.go:185]: ah nvm me :D

File: internal/cli/atlas/deployments/setup.go (hunk 1 above)
Comment (tibulca) [reply internal/cli/atlas/deployments/setup.go]: good point, done

**Review by blva — APPROVED** [review]

**Review by gssbzn — APPROVED** [review]

**Pull request merged**